frontend/components/Navbar/navbar.module.css

```
@@ -0,0 +1,63 @@
+.background {
+  padding: 0;
```

I'd recommend apply horizontal padding instead of adding `margin-left` and `margin-right` properties to the outer components.
```suggestion
  padding: 0 1rem;
```